python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst

```diff
@@ -17,34 +17,32 @@
 
 
 =====================
-Supported pandas APIs
+Supported pandas API
 =====================
 
 .. currentmodule:: pyspark.pandas
 
 The following table shows the pandas APIs that implemented or non-implemented from pandas API on
-Spark.
+Spark. Some pandas API do not implement full parameters, so the third column shows missing
+parameters for each API.
 
-Some pandas APIs do not implement full parameters, so the third column shows missing parameters for
-each API.
+* 'Y' in the second column means it's implemented including its whole parameter.
+* 'N' means it's not implemented yet.
+* 'P' means it's partially implemented with the missing of some parameters.
 
-'Y' in the second column means it's implemented including its whole parameter.
-'N' means it's not implemented yet.
-'P' means it's partially implemented with the missing of some parameters.
+All API in the list below computes the data with distributed execution except the ones that require
+the local execution by design. For example, `DataFrame.to_numpy() <https://spark.apache.org/docs/
+latest/api/python/reference/pyspark.pandas/api/pyspark.pandas.DataFrame.to_numpy.html>`__
+requires to collect the data to the driver side.
 
 If there is non-implemented pandas API or parameter you want, you can create an `Apache Spark
-JIRA <https://issues.apache.org/jira/projects/SPARK/summary>`__ to request or to contribute by your
-own.
+JIRA <https://issues.apache.org/jira/projects/SPARK/summary>`__ to request or to contribute by
+your own.
 
-The API list is updated based on the `pandas 1.3 official API
-reference <https://pandas.pydata.org/pandas-docs/version/1.3/reference/index.html>`__.
+The API list is updated based on the `latest pandas official API reference
+<https://pandas.pydata.org/docs/reference/index.html#>`__.
```

Comment:
I think we should also keep 1.3 here, and then dynamically generate this verison in followup release.

By the way, we had some discussion about this version before: https://github.com/apache/spark/pull/36095#discussion_r844611610 https://github.com/apache/spark/pull/36114#discussion_r845778366

Reply:
@Yikun  I've fixed the reference version specification of pandas to 1.3.